plugins.yaml

```
@@ -129,18 +129,18 @@ aiida-environ:
   code_home: https://github.com/environ-developers/aiida-environ
   entry_point_prefix: environ
   pip_url: git+https://github.com/environ-developers/aiida-environ
-  plugin_info: https://raw.github.com/environ-developers/aiida-environ/master/setup.json
+  plugin_info: https://raw.github.com/environ-developers/aiida-environ/master/pyproject.toml
 aiida-eon:
   code_home: https://github.com/HaoZeke/aiida-eon
   entry_point_prefix: eon
 aiida-eonclient:
   code_home: https://github.com/HaoZeke/aiida-eonclient
   entry_point_prefix: eonclient
 aiida-fenics:
-  code_home: https://github.com/sphuber/aiida-fenics/tree/master
+  code_home: https://github.com/broeder-j/aiida-fenics/tree/master
   entry_point_prefix: fenics
-  pip_url: git+https://github.com/sphuber/aiida-fenics
-  plugin_info: https://raw.github.com/sphuber/aiida-fenics/master/setup.json
+  pip_url: git+https://github.com/broeder-j/aiida-fenics
+ plugin_info: https://raw.github.com/broeder-j/aiida-fenics/master/setup.json
```

```suggestion
  plugin_info: https://raw.github.com/broeder-j/aiida-fenics/master/setup.json
```

Missing a space here ;)